_sass/custom/_card.scss

```diff
@@ -275,6 +275,24 @@
     }
 }
 
+.card-study {
+    border: 3px solid $link-color;
```

Comment:
borders should use the default '$border-color' variable. $link-color variable is reserved for a tag links.

Reply:
I'm updating the 'featured cards' in my PR to have more prominent colours besides just the default grey border.